workers/unity/Packages/io.improbable.gdk.tools/GdkToolsConfigurationWindow.cs

```diff
@@ -224,5 +248,22 @@ private void DrawCustomSnapshotDir()
                 }
             }
         }
+
+        private void Update()
+        {
+            TrySaveChanges();
+        }
+
+        private void TrySaveChanges()
+        {
+            var timeSinceLastSave = DateTime.Now - lastSaveTime;
+            if (!hasUnsavedData || timeSinceLastSave < FileSavingInterval)
+            {
+                return;
+            }
+
+            toolsConfig.Save();
+            lastSaveTime = DateTime.Now;
```

Comment:
Shouldn't we set `hasUnsavedData` to `false` after saving?

Reply:
Oh goodness. Good catch!